source/MaterialXTest/Render.cpp

```diff
@@ -1106,10 +1115,15 @@ bool getTestOptions(const std::string& optionFile, ShaderValidTestOptions& optio
 }
 
 void printRunLog(const ShaderValidProfileTimes &profileTimes, const ShaderValidTestOptions& options,
-    std::set<std::string>& usedImpls, std::ostream& profilingLog, mx::DocumentPtr dependLib,
-    mx::OslShaderGeneratorPtr oslShaderGenerator, mx::GlslShaderGeneratorPtr glslShaderGenerator
-#if defined(MATERIALX_BUILD_GEN_OGSFX)
-    , mx::OgsFxShaderGeneratorPtr ogsfxShaderGenerator
+    std::set<std::string>& usedImpls, std::ostream& profilingLog, mx::DocumentPtr dependLib
+#ifdef MATERIALX_BUILD_GEN_OSL
```

Comment:
Nice to see everything wrapped in a context. Will make refactoring at least this method easier :)